pkg/mdformatter/mdgen/mdgen.go

```diff
@@ -19,6 +19,7 @@ const (
 	infoStringKeyLang = "mdox-gen-lang"
 	infoStringKeyType = "mdox-gen-type"
 	infoStringKeyExec = "mdox-gen-exec"
+	skipExitCodeCheck = "mdox-nonzero-exit-code"
```

Comment:
I wonder... should we just don't care about exit codes? 🤔

Reply:
Also variable name is not telling the same as string

Reply:
Initially, I did think that not checking for exit codes would be better, but then user wouldn’t be able to tell which command in a md code block had an actual error and that might get formatted without the user noticing. For eg `npm —versionn` is a typo which would come up as an error if we check the exit code. However, if we skip that then something entirely different than the version number might get formatted.
Would the better option be to have an `expect-exit-code` flag which will be matched if non 0 exit code occurs?

Reply:
I think tag makes sense more like you did.

So what about by default expecting status code 0, but allow `expect-exit-code=X` to change that? 🤗

Reply:
Yup! That would be best. Implementing that along with a quick test! 😊